plugins/logs/plugin.go

```diff
@@ -892,12 +890,9 @@ func (p *Plugin) oneShot(ctx context.Context) (ok bool, err error) {
 					continue
 				}
 
-				p.mtx.Lock()
 				for _, event := range events {
 					p.encodeAndBufferEvent(event)
```

Comment:
`encodeAndBufferEvent` is called in the `Log` method as well.

Reply:
Correct. `encodeAndBufferEvent` now does it's own locking, which is why we shouldn't also lock here.